test/ectest.c

```diff
@@ -1159,6 +1159,43 @@ static int internal_curve_test_method(int n)
     return r;
 }
 
+static int group_field_test(void)
+{
+    int r = 1;
+    BIGNUM *sect521r1_field = NULL;
+    BIGNUM *sect163r2_field = NULL;
+    EC_GROUP *sect521r1_group = NULL;
+    EC_GROUP *sect163r2_group = NULL;
```

Comment:
Variable names here are confusing: it should be `secp521r1_*`

Reply:
Thanks, I missed that before. 
I changed the variables names.